src/app/qgsrasterlayerproperties.h

```diff
@@ -141,7 +141,7 @@ class APP_EXPORT QgsRasterLayerProperties : public QgsOptionsDialogBase, private
 
   signals:
     //! Emitted when changes to layer were saved to update legend
-    void refreshLegend( const QString &layerID, bool expandItem );
+    Q_DECL_DEPRECATED void refreshLegend( const QString &layerID, bool expandItem ) SIP_DEPRECATED;
```

Comment:
Actually, it's in app, didn't notice before, sorry.
I'd say kill it right away (it's neither used nor public api).